infrastructure/docker/install-dependencies/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.15
 
-RUN apk --update add --no-cache libgit2 libgit2-dev go protoc make pkgconfig git cmake yarn gettext
+RUN apk --update add --no-cache libgit2 libgit2-dev go protoc make pkgconfig git cmake pnpm gettext
```

Comment:
To do: fix this too, installing pnpm using npm

Reply:
Since the pnpm is installed globally on the builder Dockerfile, there is no need to install it in this Dockerfile. However, that pnpm add must be removed.